web/src/resize.ts

```diff
@@ -218,9 +218,24 @@ function resize_navbar_alerts(): void {
     }
 }
 
+export function resize_settings_creation_overlay(): void {
+    if ($(".two-pane-settings-creation-simplebar-container").length === 0) {
+        return;
+    }
+
+    $(".two-pane-settings-creation-simplebar-container").css(
+        "height",
+        height_of($(".two-pane-settings-container")) -
+            height_of($(".two-pane-settings-header")) -
+            height_of($(".two-pane-settings-overlay .display-type")) -
+            height_of($(".settings-sticky-footer")),
+    );
+}
```

Comment:
Just to highlight why this is brittle -- this logic is only correct to the extend that we don't adjust the set of elements. 

`container - header -footer` isn't crazy, but the `display-type` thing starts to feel like there may be other random stuff that needs to be here too over time.

Reply:
We were already sort of doing this before with calculating heights from other CSS heights, but I agree a less brittle solution would be great. If we do something else, it should work with the full #34125 solution.